### DISCUSSION TITLE
Dragabable seems is doesn work properly

### DISCUSSION THREAD

**Discussion by sotiriskast** [nolimits4web/swiper#5775] [Q&A]
var swiperBlogImg = new Swiper('.blog-crv .swiper-img',

        {
        slidesPerView: 1,
        spaceBetween: 0,
        speed: 800,
        loop: true,
        dragClickables: false,
        shortSwipes: false,
        slideToClickedSlide: true,
        preventClicksPropagation: false,
        preventClicks: false,
        touchRatio: 0,
        allowTouchMove: false,
        autoplay: {
            delay: 6000,
        },

        pagination: {
            el: '.blog-crv .controls .swiper-pagination',
            type: 'fraction',
        },

        navigation: {
            nextEl: '.blog-crv .controls .next-ctrl',
            prevEl: '.blog-crv .controls .prev-ctrl'
        },

    });

i use all of this to stop dragable the image but it doesn stop. Still drag the image to next one.

**Comment by Arhell** [comment]
update swiper
https://github.com/nolimits4web/swiper/discussions/5774#discussioncomment-2888895
